src/test/java/tests/payroll_job/LoginToSchedulerConsoleTest.java

```diff
@@ -1,24 +1,24 @@
-package tests.payrollJob_task2;
+package tests.payroll_job;
 
-import businessObjects.SchedulerConsoleUser;
-import tests.commonTest.BaseTest;
-import serviceClasses.loginService.LoginToSchedulerConsole;
-import pageObjects.payrollPages.SchedulerConsolePage;
+import business_objects.UserBO;
+import business_objects.UserFactory;
+import tests.common_test.BaseTest;
+import page_objects.payroll_pages.SchedulerConsolePage;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import static service_classes.login_service.LoginService.loginToSchedulerConsole;
+
 public class LoginToSchedulerConsoleTest extends BaseTest {
 
     private final String pageName = "Scheduler Console";
 
-    private SchedulerConsoleUser user = new SchedulerConsoleUser();
+    UserBO schedulerConsoleUser = UserFactory.createSchedulerConsoleUser();
 
     @Test(description = "Successful login to Scheduler Console", priority = 0,
             groups = {"loginToSchedulerConsole", "payroll_test"})
```

Comment:
Priority equals 0 is by default. You can remove it from current test method.

Reply:
Fixed

Reply:
Approved